### DISCUSSION TITLE
Simplified Chinese display of the list
列表的简体中文显示

### DISCUSSION THREAD

**Discussion by bilibili2233a** [community/community#28830] [Other Feature Feedback, Questions, & Ideas]
In simplified Chinese, it is generally more appropriate to use "新建", but "新" is a kind of machine-turned feeling, which is inappropriate

在简体中文中，一般以“新建”更为合适，“新”却是一种机翻的感觉，是不合适的
![Screenshot_20220809_094623.jpg](https://user-images.githubusercontent.com/99982054/183545673-bb2238b7-2694-4249-aad5-1fffa4c7b134.jpg)

**Comment by 2876egg** [comment]
PROFILE.md
